src/gerbil/gxc-main.ss

```diff
@@ -61,6 +62,10 @@
     (match rest
       ([arg . rest]
        (case arg
+         (("-V" "--version")
+          (displayln (gerbil-system-version-string))
+          (force-output)
+          (exit 0))
```

Comment:
maybe better to use `-v` for version and make verbose `-V`? That way it will be consistent with everything else, as is this is inconsistent.

Reply:
as you wish